molvis/molvisgl/src/processors/molecularrasterizer.cpp

```diff
@@ -247,31 +246,36 @@ void MolecularRasterizer::process() {
             offset += mesh->getIndices(0)->getSize();
         }
     }
-
-    std::shared_ptr<const Rasterization> rasterization =
-        std::make_shared<MolecularRasterization>(*this);
-    outport_.setData(rasterization);
 }
 
 void MolecularRasterizer::initializeResources() {
-    for (auto& item : vdwShaders_->getShaders()) {
-        configureVdWShader(item.second);
+    for (auto& [state, shader] : vdwShaders_.getShaders()) {
+        configureVdWShader(shader);
     }
-    for (auto& item : licoriceShaders_->getShaders()) {
-        configureLicoriceShader(item.second);
+    for (auto& [state, shader] : licoriceShaders_.getShaders()) {
+        configureLicoriceShader(shader);
     }
 }
 
+/*
+void SphereRasterizer::configureShader(Shader& shader) {
+    Rasterizer::configureShader(shader);
+    utilgl::addDefines(shader, labels_, periodic_, config_, clip_);
+    shader.build();
+}
+*/
```

Comment:
remove? that about periodic/config/clip?

Reply:
copied from `void SphereRasterizer::configureShader()` for reference. Should be removed.